Agile Development/Burndown Chart/script.py

```diff
@@ -0,0 +1,132 @@
+import pprint
+import requests
+import jpholiday
+import datetime
+import matplotlib.dates as mdates
+import pandas as pd
+import matplotlib.pyplot as plt
+import urllib.request
+import urllib.parse
+import json
+import datetime
+from pytz import timezone
+point_dict = {}
+# ---- #
+# init #
+# ---- #
+total_points = 0
+done = 0
+undone = 0
+BASIC = 'Basic XXXXXXXXXXXXXXX=='
+SPRINT = "Sprint1"
```

Comment:
Instead of variables in the script, can you switch to using [argparse](https://docs.python.org/3/library/argparse.html) and taking these as inputs via command line so a user doesn't need to open the script and modify values every time?

Reply:
It certainly seems easier to use that way. 
I'll give it a try!